translations/texts/objects/apex/apexconsolekeyboard/apexconsolekeyboard.object.json

```
@@ -20,7 +20,7 @@
     },
     "Texts": {
       "Eng": "A keyboard. Used for typin'.",
-      "Rus": ""
+      "Rus": "Клавиатура. Используется для набора текста."
```

```suggestion
      "Rus": "Клавиатура. Для того, чтоб печатать."
```
Немного исправил стилистику, тк это фраза новакида